### ISSUE TITLE
two different output from one variant

### ISSUE THREAD

**Issue by jungkim2** [diskin-lab-chop/AutoGVP#50]
three variants that annotate in two different ways
[error_clinvar.txt](https://github.com/diskin-lab-chop/AutoGVP/files/11909345/error_clinvar.txt)
[error_intervar.txt](https://github.com/diskin-lab-chop/AutoGVP/files/11909346/error_intervar.txt)

**Comment by naqvia** [comment]
What is the output of these variants? Can you share the resulting table, so I can take a deeper look.

**Comment by naqvia** [comment]
Can you re-run and see if this issue persists? We had made significant changes to the script and I suspect this issue may have been resolved.

**Comment by jungkim2** [comment]
Just ran, and above issue has not been resolved.

**Comment by naqvia** [comment]
I am looking to these files and manually seeing if I can spot an issue. I think I have narrowed down to where the issue may be arising from. One thing I have noticed is that all of these cases there is a tie in the clinVar calls (Conflicting interpretation.. .) which means I need to look these up in the submission files and take the latest call. Will look deeper and create a PR for this request, but just an update!
